src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs

```diff
@@ -37,6 +37,7 @@ internal class ApplicationInsightsLogger : ILogger
                 ScopeKeys.Event,
                 ScopeKeys.FunctionInvocationId,
                 ScopeKeys.FunctionName,
+                ApplicationInsightsScopeKeys.HttpRequest,
```

Comment:
Is there any other scope information that app insights adds which we might want to be cautious about?

Can we have a comment on why we are excluding this information?

Reply:
No, this is really the only thing we add from the Functions side of things. And really it's the only 'important' one to drop. I dropped all the other ones to save space because they were redundant.

Yeah, I'll add some details.